2021-08-18 12:11:40,125 INFO    MainThread:1599 [wandb_setup.py:_flush():69] setting env: {}
2021-08-18 12:11:40,125 INFO    MainThread:1599 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-08-18 12:11:40,125 INFO    MainThread:1599 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-08-18 12:11:40,125 INFO    MainThread:1599 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-18 12:11:40,125 INFO    MainThread:1599 [wandb_init.py:_log_setup():342] Logging user logs to /root/konst/ml-utils/training/edm_genre_classify/wandb/run-20210818_121140-1thkpjpx/logs/debug.log
2021-08-18 12:11:40,125 INFO    MainThread:1599 [wandb_init.py:_log_setup():343] Logging internal logs to /root/konst/ml-utils/training/edm_genre_classify/wandb/run-20210818_121140-1thkpjpx/logs/debug-internal.log
2021-08-18 12:11:40,126 INFO    MainThread:1599 [wandb_init.py:init():375] calling init triggers
2021-08-18 12:11:40,126 INFO    MainThread:1599 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {'data_path': 'melspects_general.npz', 'checkpoint_path': 'model_checkpoints/', 'model_path': '', 'batch_size': 20, 'epochs': 100, 'start_epoch': 0, 'lam': 0.04, 'patience': 2, 'factor': 0.8, 'use_wandb': True, 'lr': 0.001, 'save_at': 1, 'adjust_lr_every': 1, 'resize': 180, 'num_workers': 1}
2021-08-18 12:11:40,126 INFO    MainThread:1599 [wandb_init.py:init():424] starting backend
2021-08-18 12:11:40,126 INFO    MainThread:1599 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-18 12:11:40,142 INFO    MainThread:1599 [backend.py:ensure_launched():135] starting backend process...
2021-08-18 12:11:40,149 INFO    MainThread:1599 [backend.py:ensure_launched():140] started backend process with pid: 1617
2021-08-18 12:11:40,150 INFO    MainThread:1599 [wandb_init.py:init():429] backend started and connected
2021-08-18 12:11:40,152 INFO    MainThread:1599 [wandb_init.py:init():477] updated telemetry
2021-08-18 12:11:40,152 INFO    MainThread:1599 [wandb_init.py:init():500] communicating current version
2021-08-18 12:11:40,587 INFO    MainThread:1599 [wandb_init.py:init():505] got version response 
2021-08-18 12:11:40,587 INFO    MainThread:1599 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-08-18 12:11:40,802 INFO    MainThread:1599 [wandb_init.py:init():540] starting run threads in backend
2021-08-18 12:11:41,675 INFO    MainThread:1599 [wandb_run.py:_console_start():1584] atexit reg
2021-08-18 12:11:41,676 INFO    MainThread:1599 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-08-18 12:11:41,677 INFO    MainThread:1599 [wandb_run.py:_redirect():1463] Redirecting console.
2021-08-18 12:11:41,679 INFO    MainThread:1599 [wandb_run.py:_redirect():1519] Redirects installed.
2021-08-18 12:11:41,679 INFO    MainThread:1599 [wandb_init.py:init():565] run started, returning control to user process
2021-08-18 12:11:48,880 INFO    MainThread:1599 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 1
2021-08-18 12:11:48,880 INFO    MainThread:1599 [wandb_run.py:_restore():1526] restore
2021-08-18 12:11:51,694 INFO    MainThread:1599 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1153
  total_bytes: 1153
}

2021-08-18 12:11:51,879 INFO    MainThread:1599 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1153
  total_bytes: 1153
}

2021-08-18 12:11:52,807 INFO    MainThread:1599 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1153
  total_bytes: 6430
}

2021-08-18 12:11:52,908 INFO    MainThread:1599 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1153
  total_bytes: 6430
}

2021-08-18 12:11:53,010 INFO    MainThread:1599 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6430
  total_bytes: 6430
}

2021-08-18 12:11:53,111 INFO    MainThread:1599 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6430
  total_bytes: 6430
}

2021-08-18 12:11:53,212 INFO    MainThread:1599 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6430
  total_bytes: 6430
}

2021-08-18 12:11:53,314 INFO    MainThread:1599 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6430
  total_bytes: 6430
}

2021-08-18 12:11:53,415 INFO    MainThread:1599 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6430
  total_bytes: 6430
}

2021-08-18 12:11:53,516 INFO    MainThread:1599 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6430
  total_bytes: 6430
}

2021-08-18 12:11:53,812 INFO    MainThread:1599 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6430
  total_bytes: 6430
}

2021-08-18 12:11:53,914 INFO    MainThread:1599 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6430
  total_bytes: 6430
}

2021-08-18 12:11:55,145 INFO    MainThread:1599 [wandb_run.py:_show_files():1898] logging synced files
